src/dc/sample/ObjectScript.cls

```diff
@@ -2,12 +2,43 @@ Class dc.sample.ObjectScript
 {
 
 ClassMethod Test() As %Status
+{
+    set sc = $$$OK
+    Try {
+        set sc = ..Run() ; Python
+        set sc = ..Run(0) ; ObjectScript
+    }
+    Catch ex {
+        set sc=ex.AsStatus()
+    }
+    return sc
+}
+
+ClassMethod Run(python As %Boolean = 1) As %Status
+{
+    return:python ..Python()
+    return:'python ..ObjectScript()
+}
+
+ClassMethod ObjectScript() As %Status
 {
     set a=42 
-    write "Hello World!",!
+    write "Hello World! from ",$CLASSMETHOD("dc.sample.ObjectScript","%ClassName",1)," in ObjectScript language",!
```

Comment:
why not just `..%ClassName(1)`?

Reply:
It is just to illustrate the $CLASSMETHOD use